src/Iterator/StringIterator.php

```diff
@@ -47,4 +50,35 @@ static function (string $input, string $delimiter): Generator {
 
         $this->iterator = new IteratorIterator($callback($data, $delimiter));
     }
+
+    /**
+     * @return string
+     */
+    public function current()
```

Comment:
```suggestion

    public function current(): string
```

why not like this? Same for others where the type is known

Reply:
Done.